### DISCUSSION TITLE
Welcome to Trillek Engine Discussions!

### DISCUSSION THREAD

**Discussion by adam4813** [trillek-team/tec#198] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by milesrout** [comment]
This seems like a good idea. A lot better than Discourse!

**Reply by adam4813** [reply]
it appears to be fairly new, and I figured why not enable it.
